packages/desktop-client/src/components/App.tsx

```diff
@@ -86,14 +98,16 @@ function AppInner() {
           loadingText: t('Retrieving remote files...'),
         }),
       );
-      send('get-remote-files').then(files => {
+      await send('get-remote-files').then(files => {
```

Comment:
Both await and .then is weird. Shouldn't it just be await?

Reply:
Fair point. It previously used then and as such had a bug due to not waiting on the promise. Happy to change it fully to use await